addons/google_recaptcha/static/src/xml/recaptcha.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+<t t-name="recaptcha.legal_terms">
+    <small class="o_recaptcha_legal_terms">
```

Comment:
can we make it still more discret, link not in cta color, align to input, text-muted, and editable and hiddable via the form builder ?
For people that don't want to respect these guidelines. (static xml is really not easy to customize ;))

Reply:
![image](https://user-images.githubusercontent.com/4848797/83129366-00365a00-a0dd-11ea-9a33-633cd7f2cc36.png)

Reply:
I can make it editable in the editor by not removing it when entering edit mode.
A little + could be that on save verify if the text is different from the original template if yes keep it in the page else remove it so if recaptcha is disabled the text is no longer added.

Reply:
Or just use a t-if=«captchaenable » and wrap text inside  ?

Reply:
to discuss tomorrow

Reply:
@JKE-be `t-if` would work on a qweb template but not on a dropped snippet.

Reply:
in js in this case. 
hide by default once website_form installed
and make it visible once you checked that token to load js script ?

Reply:
Or keep it simple:
t-if in qweb...

as well it was enable before the drop and it works
as well he add the token after the drop and can always add himself the warning manually. It is just a small text. (and probably that most of the users will remove id ;))